apps/examples/testcase/ta_tc/device_management/itc/itc_dm_lwm2m.c

```diff
@@ -160,7 +160,7 @@ static void itc_dm_lwm2m_get_server_port_p(void)
 */
 static void itc_dm_lwm2m_get_client_state_p(void)
 {
-	int8_t state = -1;
+	dm_lwm2m_client_state_e state = -1;
 	int ret = dm_lwm2m_start_client(&test_data_itc);
```

Comment:
why don't you use dm_error_e instead of int as a return value type?
You also need to modify the return value types of all DM apis, that is, dm_error_e.

Reply:
dm_lwm2m_client_state_e has no value of -1.